backend/prompt_studio/prompt_studio_output_manager/serializers.py

```diff
@@ -1,9 +1,28 @@
+import logging
+
+from usage.helper import UsageHelper
+
 from backend.serializers import AuditSerializer
 
 from .models import PromptStudioOutputManager
 
+logger = logging.getLogger(__name__)
+
 
 class PromptStudioOutputSerializer(AuditSerializer):
     class Meta:
         model = PromptStudioOutputManager
         fields = "__all__"
+
+    def to_representation(self, instance):
+        data = super().to_representation(instance)
+        try:
+            token_usage = UsageHelper.get_aggregated_token_count(instance.run_id)
+        except Exception as e:
```

Comment:
@tahierhussain Catching specific exceptions over `Exception` will surface error better to the user.

Reply:
@harini-venkataraman That is one pending item that we need to fix everywhere in the code.

Reply:
@Deepak-Kesavan @tahierhussain but let's make sure we don't follow this incorrect practice for all new code we add

Reply:
@harini-venkataraman @Deepak-Kesavan We're only logging the error and not raising exception. The error is handled gracefully as we're returning an empty object instead. In this case, I don't think it will be required to catch specific exceptions. Please let me know you opinion.

Reply:
@tahierhussain where are we re-raising these exceptions? It's only being caught and logged I think

Reply:
@chandrasekharan-zipstack Yes. We're not raising the exception. That's expected. We don't want to raise an exception, if we're unable to get the persisted token usage data for the prompt output API response.